### PR TITLE
Update test-owners with new tests, add catch-all assignment to test-infra team.

### PR DESCRIPTION
**Pull request by rmmh**
We will triage any additional failures, since they're more likely to be infra related. If they're not, they can always be reassigned (and the owners list can be updated!)

/cc @kubernetes/test-infra-maintainers

### CODE REVIEW

**Comment by gmarek** [comment]
Do we track load on people somewhere?

**Comment by rmmh** [comment]
If you run hack/update_owners.py, it outputs load as part of the reassignments. Each person currently has ~11 test assigned, but it's not a very fair measurement given how some test are much flakier than others.

**Comment by gmarek** [comment]
So I guess the answer is "no". Do we plan to address that? @fejta

File: test/test_owners.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name,owner,auto-assigned
+$DEFAULT,rmmh/spxtr/ixdy/apelisse/fejta,0
```
Comment (ixdy) [inline test/test_owners.csv]: does automation handle this properly?

File: hack/update_owners.py [hunk 2]
```diff
@@ -73,28 +73,30 @@ def get_maintainers():
...
+           "sttts", "swagiaal", "thockin", "timothysc", "timstclair", "tmrts",
+           "vishh", "vulpecula", "wojtek-t", "xiang90", "yifan-gu", "yujuhong",
+           "zmerlynn"}
     return sorted(ret - SKIP_MAINTAINERS)
 
 
 def main():
     test_history = get_test_history()
     test_names = sorted(set(map(normalize, test_history['test_names'])))
+    test_names.append('$DEFAULT')
```
Comment (ixdy) [inline hack/update_owners.py]: is $DEFAULT a magic string or something?

File: test/test_owners.csv (hunk 1 above)
Comment (rmmh) [inline test/test_owners.csv]: It does with the PR I have outstanding in contrib/

File: hack/update_owners.py (hunk 2 above)
Comment (rmmh) [inline hack/update_owners.py]: Yes, it's the catch-all fallback name. See the related PR in contrib/.

Comment (ixdy) [inline hack/update_owners.py]: I agree with @fejta that the $ is confusing.

also, the case doesn't match that PR - does this matter?

**Comment by rmmh** [comment]
Updated with "default" as the fallback key instead of "$default".

File: hack/update_owners.py [hunk 3]
```diff
@@ -61,7 +61,9 @@ def write_owners(fname, owners):
     with open(fname, 'w') as f:
         out = csv.writer(f, lineterminator='\n')
         out.writerow(['name', 'owner', 'auto-assigned'])
-        for name, (owner, random_assignment) in sorted(owners.items()):
+        sort_key = lambda (k, v): (k != 'DEFAULT', k)
```
Comment (ixdy) [inline hack/update_owners.py]: minor suggestion: make 'DEFAULT' a global constant?

Comment (ixdy) [inline hack/update_owners.py]: I'm also a little confused why you're using a `(bool, string)` key for sorting.

Comment (rmmh) [inline hack/update_owners.py]: It's just to put 'DEFAULT' at the top of the list.

Comment (ixdy) [inline hack/update_owners.py]: that's a cute and disgusting hack. maybe at least document it?

Comment (rmmh) [inline hack/update_owners.py]: Documented.

**Pull request merged**